### DISCUSSION TITLE
Tip 1

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-script-sample#89] [Tip]
1.퍼블릭으로 추가한 변수가 인스펙터에 나오지 않을때는 스크립트 내에서 그 변수가 호출되었는지 살펴보자
-변환과정에서 호출되지 않는 변수 "never use..." 애들은 그냥 부르지 않는것 같다.

2.맥을 사용하자, 인스팩터가 바로 업데이트 된다.

3.게임 도중에는 인스팩터의 값을 바꿀 수 없다.
-게임도중 인스팩터의 값을 바꾸고 그 변수를 호출하면 오류를 뱉어준다. (이미 변환이 끝나서 그런것 같은데 잘은 모르겠네요)

4.퍼블리쉬 버튼 오른쪽에 "오픈월드셋팅"에 들어가면 가로버전세로버전을 정할 수 있다.

5.프로젝트 셋팅 -> 플레이어 -> Active Input Handling 을 Both로 바꾸어주자

6.프로젝트 셋팅 -> 퀄리티에서 택스쳐 퀄리티를 낮추면 QR코드가 깨진다

7.프로젝트 셋팅에 피직스의 콜리전 메트릭스를 충돌할 놈들끼리만 켜주자 (프레임수가 확 올라간다)

오피셜은 아닙니다. 개발자들끼리 정보 공유해요.

**Comment by xowjd913** [comment]
1. 어느 순간부터 ZepetoScriptContext에 달려있던 컴포넌트가 사라져 BreakingPoint를 찍어볼수가없다.
    - Zepeto 측에서 이슈 수정 해줘야 할듯 싶네요.
    
2. Resources.Load를 통해 Resources 폴더 내 Text / Binary 파일을 Unity에서 로드 하듯 쓸수있음 (속도 보장 못함)
    - byte[]는 console log로 찍어보면 Undefined로 나옴 (Breaking Point 로 찍어서 내부값 확인해봐야됨)
    - string 값들은 잘읽어옴

**Comment by jeejul1010** [comment]
혹시 씬전환 모바일에서도 잘 이루어지나요?? 저는 제페토에서 자동생성된 로딩씬이 중간에 삽입돼서 게임씬으로 진입이 안됩니다. 계속 제페토 로딩씬에 멈춰있고 게임씬 진입 전 2씬 정도 만들었는데 맨 처음 씬만 보이고 바로 제페토 로딩씬으로 이동 후 화면이 거기 멈춰있네요. 에디터에서는 문제없이 씬 전환이 잘 이루어집니다. 도움 주시면 감사하겠습니다. 빌드세팅에서 모든 씬 추가했고 코드는 간단하게 LoadScene 사용해서 다음 인덱스를 파라미터로 넘겨줬는데 도움주실 수 있으면 감사하겠습니다.

**Comment by x4cf** [comment]
저는 씬 로드를 사용하기 않아서 잘 모르지만 씬 이동전에 서버 연결을 끊고 다음 씬에서 다시 연결해야되는것같아보입니다.

**Comment by x4cf** [comment]
지금은 가이드가 워낙 부족해서 잘 작동하는 기능들을 이용해서 월드를 만드는게 가장 좋아보입니다.

**Comment by jeejul1010** [comment]
그래야겠네요ㅠㅠ 답변 감사합니다😊

2021년 11월 19일 (금) 오후 1:43, x4cf ***@***.***>님이 작성:

> 지금은 가이드가 워낙 부족해서 잘 작동하는 기능들을 이용해서 월드를 만드는게 가장 좋아보입니다.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/naverz/zepeto-script-sample/issues/73#issuecomment-973730593>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI4WHUIFAAE7A5TJHURJ6DDUMXIW3ANCNFSM5ILAVN6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
hellokitty<3er

**Comment by smartbh** [comment]
혹시 제페토 모바일 테스트 잘 돌아가시나요? 저는 방화벽 해제까지 하고 해봤는데 여전히 로딩이 안되던데 무슨 이유인지 이해가 안가네요 #69

**Comment by x4cf** [comment]
모바일 테스트 잘 돌아갑니다. 이곳에서 더 자세히 물어보시는것도 좋은 방법입니다 :)

코드 : Me793a2
https://open.kakao.com/o/gNaIjUKd
